services/ingestion/etl/just/csv/src/lib/transform.js

```diff
@@ -0,0 +1,166 @@
+// @flow
+
+import getCountryCode from '../../../../../helpers/getCountryCode';
+
+/*
+ * Transform message (JUST CSV)
+ */
+
+import type { Project } from '../../../../_types/Project';
+
+const getFundingArea = () => [];
```

Comment:
If there's no actual operation on a field, you don't really need to make a helper. Though, if you prefer to keep helpers, please include docblocks to them, so that they are included in the documentation.